po/pl.po

```diff
@@ -5361,7 +5361,7 @@ msgstr "Opcje GnuCash"
 #: gnucash/gnome/gnc-plugin-page-report.cpp:1892
 #: gnucash/report/reports/standard/invoice.scm:849
 msgid "Printable Invoice"
-msgstr "Drukowalna faktura"
+msgstr "Faktura (do druku)"
```

Comment:
@6e2f61 IMHO it is a bad idea to use brackets in the glossary if they do not exist in theenglish messaage.